File: app/src/main/java/com/example/dagger_hilt/data/datasource/web/models/MovieModel.java
package com.example.dagger_hilt.data.datasource.web.models;

import com.google.gson.annotations.SerializedName;

public class MovieModel {
    @SerializedName("id")
    int id;

    @SerializedName("title")
    String title;

    @SerializedName("original_title")
    String originalTitle;

    @SerializedName("overview")
    String overview;

    @SerializedName("poster_path")
    String posterPath;

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getTitle() {
        return title;
    }

    public String getOriginalTitle() {
        return originalTitle;
    }

    public String getOverview() {
        return overview;
    }

    public String getPosterPath() {
        return posterPath;
    }

}

File: app/src/main/java/com/example/gob_fact/data/datasource/web/models/MovieModel.kt
package com.example.gob_fact.data.datasource.web.models

import com.google.gson.annotations.SerializedName

data class MovieModel (
    @SerializedName("id")
    var id: Int,

    @SerializedName("title")
    var title: String,

    @SerializedName("original_title")
    var originalTitle: String,

    @SerializedName("overview")
    var overview: String,

    @SerializedName("poster_path")
    var posterPath: String,
)